rosdep/base.yaml

```diff
@@ -2891,7 +2897,7 @@ libvtk-qt:
     stretch: [libvtk6-qt-dev]
     wheezy: [libvtk5-qt4-dev]
   fedora: [vtk-qt]
-  gentoo: [sci-libs/vtk]
+  gentoo: ['sci-libs/vtk[qt4]'] #qt5 is available too
```

Comment:
no, the rosdep key is ambiguous because doesn't specify the version, is better to fix the rosdep key and leaving qt4 for now